packages/web/package.json

```diff
@@ -41,6 +41,8 @@
     "@keplr-wallet/wc-client": "0.10.24-ibc.go.v7.hot.fix",
     "@metamask/onboarding": "^1.0.1",
     "@next/bundle-analyzer": "^12.1.6",
+    "@notifi-network/notifi-frontend-client": "0.75.2-alpha.0",
+    "@notifi-network/notifi-react-card": "0.75.2-alpha.0",
```

Comment:
Please use next/dynamic to import these modules dynamically when the user first opens the notifications modal to avoid further bloating the main bundle. We have many examples of this in the app.

Reply:
Added dynamic imports